src/components/report/TermsAndConditionsDialog.jsx

```diff
@@ -1,15 +1,13 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
 import DialogContent from '@material-ui/core/DialogContent';
-import { selectTermsAndConditions } from '../../modules/site/selectors';
 import Text from '../Text';
 import StandardDialog from '../StandardDialog';
 
 export default function TermsAndConditionsDialog({
   visible,
   onClose,
 }) {
-  const termsAndConditions = useSelector(selectTermsAndConditions);
+  const termsAndConditions = 'These are our terms: liberty or death!';
```

Comment:
Lol... I imagine that this does not need to go into en.json?

Reply:
No.. I pinged Tanya about actually creating these and then we can incorporate them correctly